guideline-app/app/ui/components/left-navigation/LeftNavigation.js

```diff
@@ -1,83 +1,88 @@
-    <div className="leftNavigation">
-        <nav aria-labelledby="left-navigation-title">
-            <Systemtittel id="left-navigation-title"><TitleByRoute routes={routeConfig} /></Systemtittel>
-            <ul className="nav-list">
-                {
-                    props.routes && props.routes.map((item) => (
-                        <li
-                            key={item.title}
-                            className={classnames({ divider: item.path === '/components/other' })}
-                        >
-                            <NavLink
-                                activeClassName="active"
-                                to={item.path}
-                            >
-                                { item.title }
-                                {
-                                    isBeta(item)
-                                    && <EtikettFokus><Undertekst>Beta</Undertekst></EtikettFokus>
-                                }
-                                {
-                                    isStyle(item)
-                                    && <EtikettInfo><Undertekst>CSS</Undertekst></EtikettInfo>
-                                }
-                            </NavLink>
-                        </li>
-                    ))
-                }
-            </ul>
-            {
-                props.routes && props.routes[0].path.indexOf('/components') === 0
-                && (
-                    <React.Fragment>
-                        <div className="contribute-promo">
-                            <Undertittel>Noe du savner?</Undertittel>
-                            <LenkepanelBase href="https://github.com/navikt/nav-frontend-moduler" border>
-                                <Normaltekst className="lenkepanel__heading">
-                                    Bidra med nye komponenter på Github
-                                </Normaltekst>
-                            </LenkepanelBase>
-                        </div>
-                    </React.Fragment>
-                )
-            }
-        </nav>
-    </div>
+  <div className="leftNavigation">
+    <nav aria-labelledby="left-navigation-title">
+      <Systemtittel id="left-navigation-title">
+        <TitleByRoute routes={routeConfig} />
+      </Systemtittel>
+      <ul className="nav-list">
+        {props.routes &&
+          props.routes.map((item) => (
+            <li
+              key={item.title}
+              className={classnames({
+                divider: item.path === "/components/other",
+              })}
+            >
+              <NavLink activeClassName="active" to={item.path}>
+                {item.title}
+                {isBeta(item) && (
+                  <EtikettFokus>
+                    <Undertekst>Beta</Undertekst>
+                  </EtikettFokus>
+                )}
+                {isStyle(item) && (
+                  <EtikettInfo>
+                    <Undertekst>CSS</Undertekst>
```

Comment:
Ser ut som denne linjen har mistet to stiler. "Tabell CSS" på nettsiden har ikke mellomrom lengre.